src/applications/vaos/newAppointmentFlow.js

```diff
@@ -0,0 +1,48 @@
+import { getFormData } from './utils/selectors';
+
+const AUDIOLOGY = '203';
+
+export default {
+  home: {
+    url: '/',
+  },
+  typeOfCare: {
+    url: '/new-appointment',
+    next: 'contactInfo',
+    // async next(state) {
```

Comment:
I'll remove this probably, but it's an example of the kind of thing we could do in here.